packages/peregrine/lib/talons/CategoryTree/useCategoryTree.js

```diff
@@ -70,9 +70,10 @@ export const useCategoryTree = props => {
         }
 
         children.map(category => {
-            const isLeaf = !parseInt(category.children_count);
-
-            childCategories.set(category.id, { category, isLeaf });
+            if (category.include_in_menu) {
```

Comment:
I'd guess that you need to now test this case.

Reply:
@sirugh  I added the test, but still getting the same result. Apparently Coverrals is referring to a miss branch? 
![image](https://user-images.githubusercontent.com/12599508/120899298-65346200-c605-11eb-94d6-881d29818a4b.png)

And the other issue a missing label in the PR apparently. But I don't see a button to add label on this PR...

Reply:
@sirugh is there anything I could do? I've seen you added a version label, but that error didn't go away.

Reply:
Don't worry about the label warning, as for the coveralls error I'll have to check but it should highlight the missing coverage if you click the file.